src/main/java/com/nukkitx/proxypass/ProxyPass.java

```diff
@@ -58,6 +58,7 @@
     private final AtomicBoolean running = new AtomicBoolean(true);
     private BedrockServer bedrockServer;
     private final Set<BedrockClient> clients = Collections.newSetFromMap(new ConcurrentHashMap<>());
+    private Integer maxClients = 0;
```

Comment:
Missed another here

Reply:
Thank you for the hint, changed it. What is the reasoning why you want to use primitves only?